### PR TITLE
[CRIMAPP-697] Display absent capital records

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change
Display empty savings, national savings certificates, assets and investments in the cya, review, completed and returned application pages if the question was asked in apply (i.e. for either way, indictable, already in crown court cases). 

Also persist national saving certificate 'none' entry in the form

## Link to relevant ticket
[CRIMAPP-697](https://dsdmoj.atlassian.net/browse/CRIMAPP-697)

### After changes:

**Check your answer screen when asked for asset, savings, national saving certificate and investment info**

Screenshot 2024-04-09 at 14 54 03
Screenshot 2024-04-09 at 14 54 08

**Review application screen**

Screenshot 2024-04-09 at 14 53 39

**Completed application screen**

Screenshot 2024-04-09 at 14 52 27

**Check your answer screen when not asked for asset, savings, national saving certificate and investment info**

Screenshot 2024-04-09 at 14 54 38

**Persist national savings certificate value**

https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/e83270e7-05aa-46c4-b3b2-782257d98b8c

## How to manually test the feature

Start a means tested application with a case type of either way, indictable or already in crown court 
Enter a nino 
Select no benefits
Fill in the information until you reach the capital screens (starting with assets)
Select no to assets, savings, national savings certificate and investments
When you get to the cya page, confirm that you can see the assets, savings, national savings certificate and investments sections 
Confirm that the design matches the above 
Check the links take you to the 'which ___ does the client have' screen 
Then navigate to the review the application screen and confirm the same
Then submit the application and navigate to the completed application page and confirm the design is the same and no change links appear 

Secondly, start a means tested application with a case type that is **not** either way, indictable or already in crown court 
Follow the same steps as above until you reach the capital screens
You should not be asked for asset, savings, national saving certificate and investment info
Answer the questions until you reach the cya page
Confirm that the design matches the above screenshot and that the assets, savings, national savings certificate and investments sections are not visible
Then navigate to the review the application screen and confirm the same
Then submit the application and navigate to the completed application page and confirm the design is the same

[CRIMAPP-697]: https://dsdmoj.atlassian.net/browse/CRIMAPP-697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/services/adapters/structs/capital_details_spec.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
```
Comment (hiboabd) [inline spec/services/adapters/structs/capital_details_spec.rb:1]: I didn't change anything in this struct but suddenly rubocop noticed there was no code coverage

File: app/forms/steps/capital/has_national_savings_certificates_form.rb [hunk 2]
```diff
@@ -13,6 +13,8 @@ class HasNationalSavingsCertificatesForm < Steps::BaseFormObject
       private
 
       def persist!
+        capital.update(attributes)
```
Comment (hiboabd) [inline app/forms/steps/capital/has_national_savings_certificates_form.rb:16]: To persist the yes no selection

File: config/locales/en/summary.yml [hunk 3]
```diff
@@ -604,6 +616,10 @@ en:
       percentage_owned:
         question: What percentage of the %{asset} do they own?
         absence_answer: *absence_none
+      has_assets:
+        question: Which assets does your client own or part-own inside or outside the UK?
+        answers:
+          'no': My client does not own any of these assets
```
Comment (arthurashman) [inline config/locales/en/summary.yml:622]: Designs just say 'None'

File: config/locales/en/summary.yml [hunk 4]
```diff
@@ -540,6 +544,10 @@ en:
         question: Whose name is the investment in?
         answers: *OWNERSHIP_TYPE 
         absence_answer: ''
+      has_investments:
+        question: Which investments does your client have inside or outside the UK?
+        answers:
+          'no': My client does not have any of these investments
```
Comment (arthurashman) [inline config/locales/en/summary.yml:550]: Designs just say 'None'

File: config/locales/en/summary.yml [hunk 5]
```diff
@@ -515,7 +515,11 @@ en:
         absence_answer: '' 
       sort_code:
         question: What is the sort code or branch name?
-        absence_answer: '' 
+        absence_answer: ''
+      has_capital_savings:
+        question: Which savings does your client have inside or outside the UK?
+        answers:
+          'no': My client does not have any of these savings
```
Comment (arthurashman) [inline config/locales/en/summary.yml:522]: Designs just say 'None'

**Review by arthurashman — APPROVED** [review]
LGTM

**Pull request merged**